beets/util/artresizer.py

```diff
@@ -240,10 +259,15 @@ def convert_format(self, source, target, deinterlaced):
             return source
 
     @property
-    def can_compare(self):
+    def can_compare(self) -> bool:
         return self.version() > (6, 8, 7)
 
-    def compare(self, im1, im2, compare_threshold):
+    def compare(
+            self,
+            im1: Image,
```

Comment:
Looks like leaving these type hints like this causes tox tests to fail; I would have expected the hints to be ignored, but apparently they're not.

Typing these as `'Image'` seems to fix this.

Reply:
There's steps that need to be taken to fix circular imports. I do that as the last thing because it's a gigantic pain. Don't worry about failing tests; at this stage the typings are what matter.